packages/ckeditor5-package-generator/lib/templates/js/README.md

```diff
@@ -0,0 +1,152 @@
+<%= packageName %>
+<% print( '='.repeat( packageName.length ) ) %>
+
+This package was created by the [ckeditor5-package-generator](https://www.npmjs.com/package/ckeditor5-package-generator) package.
+
+## Table of contents
+
+* [Developing the package](#developing-the-package)
+* [Available scripts](#available-scripts)
+  * [`start`](#start)
+  * [`test`](#test)
+  * [`lint`](#lint)
+  * [`stylelint`](#stylelint)
+  * [`build:dist`](#builddist)
+  * [`translations:collect`](#translationscollect)
+  * [`translations:download`](#translationsdownload)
+  * [`translations:upload`](#translationsupload)
+* [License](#license)
+
+## Developing the package
+
+To read about the CKEditor 5 Framework, visit the [CKEditor 5 Framework documentation](https://ckeditor.com/docs/ckeditor5/latest/framework/index.html).
+
+## Available scripts
+NPM scripts are a convenient way to provide commands in a project. They are defined in the `package.json` file and shared with other people contributing to the project. It ensures that developers use the same command with the same options (flags).
+
+All the scripts can be executed by running `<%= packageManager %> run <script>`. Pre and post commands with matching names will be run for those as well.
+
+The following scripts are available in the package.
+
+### `start`
+
+Starts an HTTP server with the live-reload mechanism that allows previewing and testing of plugins available in the package.
+
+When the server has been started, the default browser will open the developer sample. This can be disabled by passing the `--no-open` option to that command.
```

Comment:
```suggestion
When the server starts, the default browser will open the developer sample. This can be disabled by passing the `--no-open` option to that command.
```